cmd/srcd/cmd/sql.go

```diff
@@ -40,7 +40,12 @@ var sqlCmd = &cobra.Command{
 		}
 		query := args[0]
 
-		c, err := daemon.Client()
+		workdir, err := os.Getwd()
+		if err != nil {
+			logrus.Fatalf("could not get working directory: %s", err)
+		}
+
+		c, err := daemon.Client(workdir)
```

Comment:
I feel like you could create a new function daemon.ClientOnPwd and move the code calling os.Getwd into it

Reply:
Actually, Client never needed workdir cause the only place you need a workdir different than pwd is Start, so I'm gonna remove all that